index.md

```diff
@@ -22,160 +22,877 @@ The following projects are included as part of the Data and Visualization SDK:
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [Ascent][Ascent]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="gray" markdown="span">([^1])</td><!-- CUDA -->
+      <td class="gray" markdown="span">([^1])</td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [Cinema][Cinema]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="gray"></td><!-- CUDA -->
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [ParaView][ParaView]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="green" markdown="span">([^2])</td><!-- CUDA -->
+      <td class="gray" markdown="span">([^3])</td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [SENSEI][SENSEI]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="gray"></td><!-- CUDA -->
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [VisIt][VisIt]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="gray"></td><!-- CUDA -->
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [VTK-m][VTK-m]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="green"></td><!-- CUDA -->
+      <td class="green"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [SZ][SZ]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="gray"></td><!-- CUDA -->
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td markdown="span">
+        [ZFP][ZFP]
+      </td>
+      <td class="green"></td><!-- CPU -->
+      <td class="green"></td><!-- CUDA -->
+      <td class="gray"></td><!-- ROCm -->
+    </tr>
+    <tr>
+      <td style="background-color: #373737; color: white">Legend</td>
+      <td colspan="3">
+        <table class="status_table_legend">
+          <tr>
+            <td class="green" style="color: white">Available</td>
+            <td class="gray" style="color: white">Not Available</td>
+          </tr>
+        </table>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+[^1]: Ascent: the lastest available verison in the spack package depends on an
+      older version of VTK-m which doesn't properly implement CUDA and ROCm.
+      This creates a conflict with being able to enable the latest versions of
+      all packages with GPU features enabled.  The most recent upstream Ascent
+      release uses an appropriately newer version of VTK-m but it has not yet
+      made it to the spack package.
+
+[^2]: ParaView: while CUDA can be enabled in ParaView it requires some recent
+      patches to the `master` branch to work correctly.  We are iterating with the
+      ParaView team to get this enabled in Spack.
+
+[^3]: ParaView: ROCm support will be available in the 5.11.0 release.
+
+## ECP SDK Support
```

Comment:
TODO(svenevs): I think I should be able to discern the correct values for this table by inspecting the current `ecp-data-vis-sdk/package.py`.